aptos-move/framework/aptos-framework/sources/system_addresses.move

```diff
@@ -29,7 +29,7 @@ module aptos_framework::system_addresses {
     }
 
     public fun is_aptos_framework_address(addr: address): bool {
-        addr == @aptos_framework
+        addr == @aptos_framework || addr == @0x3
```

Comment:
This doesn't feel right to me. Why group these together? Setters in token that need to check the signer should assert against 0x3 distinctly, not 0x1.

Reply:
right, this should be `is_aptos_token_address` or something

or could be a new function `is_aptos_governance_address`

Reply:
updated